ui/src/apps/statement/components/StatementPlansTable.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Table } from 'antd'
+
+import { StatementPlan } from './statement-types'
+import { useTranslation } from 'react-i18next'
+
+export default function StatementPlansTable({
+  plans,
+}: {
+  plans: StatementPlan[]
+}) {
+  const { t } = useTranslation()
+
+  const columns = [
+    {
+      title: t('statement.plan.plan'),
+      dataIndex: 'plan',
+      key: 'plan',
+      render: (text, _plan) => {
+        return (
+          <pre>
+            <code style={{ whiteSpace: 'pre-wrap' }}>{text}</code>
```

Comment:
Can we parse the plan to make it prettier? (like rendering in a table)

Reply:
yep, we need to polish the UI later, we can put it to UCP.

Reply:
I decide to do it this time.